x-pack/plugins/security/server/prompt_page.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// @ts-expect-error no definitions in component folder
+import { EuiEmptyPrompt } from '@elastic/eui/lib/components/empty_prompt';
+// @ts-expect-error no definitions in component folder
+import { icon as EuiIconAlert } from '@elastic/eui/lib/components/icon/assets/alert';
+// @ts-expect-error no definitions in component folder
+import { appendIconComponentCache } from '@elastic/eui/lib/components/icon/icon';
+// @ts-expect-error no definitions in component folder
+import { EuiPage, EuiPageBody, EuiPageContent } from '@elastic/eui/lib/components/page';
+import type { ReactNode } from 'react';
+import React from 'react';
+
+import { i18n } from '@kbn/i18n';
+import { I18nProvider } from '@kbn/i18n/react';
+import * as UiSharedDeps from '@kbn/ui-shared-deps';
+import type { IBasePath } from 'src/core/server';
+
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { Fonts } from '../../../../src/core/server/rendering/views/fonts';
+
+// Preload the alert icon used by `EuiEmptyPrompt` to ensure that it's loaded
+// in advance the first time this page is rendered server-side. If not, the
+// icon svg wouldn't contain any paths the first time the page was rendered.
+appendIconComponentCache({
+  alert: EuiIconAlert,
+});
+
+interface Props {
+  buildNumber: number;
+  basePath: IBasePath;
+  scriptPaths?: string[];
+  title: string;
+  body: ReactNode;
+  actions: ReactNode;
+}
+
+export function PromptPage({
+  basePath,
+  buildNumber,
+  scriptPaths = [],
+  title,
+  body,
+  actions,
+}: Props) {
+  const uiPublicURL = `${basePath.serverBasePath}/ui`;
+  const regularBundlePath = `${basePath.serverBasePath}/${buildNumber}/bundles`;
+  const styleSheetPaths = [
+    `${regularBundlePath}/kbn-ui-shared-deps/${UiSharedDeps.baseCssDistFilename}`,
+    `${regularBundlePath}/kbn-ui-shared-deps/${UiSharedDeps.lightCssDistFilename}`,
+    `${basePath.serverBasePath}/node_modules/@kbn/ui-framework/dist/kui_light.css`,
+    `${basePath.serverBasePath}/ui/legacy_light_theme.css`,
+  ];
+
+  return (
+    <html lang={i18n.getLocale()}>
+      <head>
+        <title>{title}</title>
```

Comment:
I think the title should just say `Elastic`, or at least start with `Elastic` so that these tabs are consistent with the rest of Kibana. What do you think?

Reply:
Yep, I agree, let's use `Elastic`, thanks!